console/console-init/ui/src/modules/connection/components/ConnectionsToolbar/ConnectionsToolbar.tsx

```diff
@@ -70,11 +70,11 @@ const ConnectionsToolbar: React.FunctionComponent<IConnectionsToolbarProps &
   ];
   const dropdownItems = [
     <DropdownItem
-      id="cl-filter-dropdown-item-closeAll"
       key="close-all"
       value="closeAll"
       component="button"
       isDisabled={selectedConnections.length <= 0}
+      id="connect-toolbar-close-selected-dropdown"
```

Comment:
replace connect with connections for all file. This is  dropdwonitem not dropdown, so use dropdownitem for dropdwonitem. It shpould be 
id="connections-toolbar-close-selected-dropdownitem"
Please follow same for other ids.

Reply:
Implemented change.